src/shared/serviceidresolver/serviceidentity/serviceidentity.go

```diff
@@ -25,34 +25,34 @@ type ServiceIdentity struct {
 const KindService = "Service"
 const KindOtterizeLegacy = "OttrLegacy"
 
-func (si *ServiceIdentity) GetFormattedOtterizeIdentityWithoutKind() string {
+func (si ServiceIdentity) GetFormattedOtterizeIdentityWithoutKind() string {
```

Comment:
Fixes an issue where this couldn't be called in some context because the object wasn't a pointer